portfolio/src/main/webapp/script.js

```diff
@@ -40,21 +40,26 @@ function addRandomStock() {
 }
 
 // Fetches comments from the server and adds them to the DOM.
-function LoadComments() {
-  const max = getMax();
-  fetch('/data?numCommentsToLoad=' + max).then(response => response.json()).then((comments) => {
-  const commentListElement = document.getElementById('comment-list');
-  commentListElement.innerHTML = ' ';
-  comments.forEach((comment) => {
-  commentListElement.appendChild(createListElement(comment));
+async function LoadComments() {
+  const loggedIn = await isLoggedIn();
+  if(loggedIn) {
```

Comment:
small nit, there should be a space after `if`

Reply:
addressed